qiime/sdk/tests/test_visualization.py

```diff
@@ -385,7 +385,7 @@ def get_abs_path(rel):
 
     def test_peek(self):
```

Comment:
Maybe we should add another test that asserts that the warning is raised when saving with the `qza` extension?